### PR TITLE
Add form section

### PR DESCRIPTION
**Pull request by stenlisuryadinata**
Hi! I'm sending a pull request for the review of my Mobile Portfolio Version with Form included.

In this updated version we have included a "Contact Me" section with a contact form.
Give style using flex position
Include Form doing Pair Programming
Added acknowledgment of my coding partner in my README file.

Thank you in advance for your feedback!

Added form and given style to h2, paragraph and containers. Also included background

### CODE REVIEW

**Comment by stenlisuryadinata** [comment]
add a contact form and finish the mobile version

the required work : 
- add the contact form to my portfolio and the portfolio of my coding partner
- guide my users' experience in a way that helps them fill in the form without frustration

according to the below rule: 
-Name field accepts up to 30 chars.
-The email field accepts only emails.
-The message field accepts up to 500 chars.
-All fields have placeholder text.
-All fields have the correct type.
-A user cannot submit the form if one or more of the fields is empty.
-Collect data by using the [Formspree service](https://formspree.io/html).
-the elements on the page you should use Flexbox.

in the following task:
- create a contact form.
- follow the given instructions.

File: index.html [hunk 1]
```diff
@@ -214,7 +214,25 @@ <h3>Skills</h3>
...
+      <input class="contact-me-name" type="text" name="name" placeholder="Hernán" required maxlength="30" minlength="5"/>
+      <input class="contact-me-email" type="email" name="email" placeholder="hernanguemes@hotmail.com" name="contact-email-address" required>
+      </label>
+      <textarea class="contact-me-textarea" placeholder="Write your message here" required cols="30" rows="8" maxlength="500" minlength="5"></textarea>
+      <div class="action-button">
+        <button class="contact-me-submit-button" type="submit">Get in touch</button>
+      </div> 
+      <span class="closing-line"><img src="Images/Shape.png"></span>   
+    </form>  
+    </div>
```
Comment (bobb-Rob) [inline index.html:235]: Well done on this beautiful project.  Just a few things will make this project even better 👇🏾 

- [ ] Looking at the image below, the input element currently has a border as highlighted in the image. Please ensure that this border matches the Figma design. It appears that this input element should not have a border or outline at all. Kindly fix this.

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @stenlisuryadinata, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (HFG43) [reply index.html:235]: Hi there,
Thanks for the feedback! I'm resending the code for your review with the changes requested applied.
-set outline inside the form to none

Thank you in advance!

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @HFG43  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Good design 👍 
✔ Descriptive PR 👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**